src/app/components/pages/Market.jsx

```diff
@@ -26,6 +27,7 @@ class Market extends React.Component {
     static propTypes = {
         orderbook: React.PropTypes.object,
         open_orders: React.PropTypes.array,
+        open_orders_sort: React.PropTypes.object,
```

Comment:
Do we want to start describing shape in our proptypes? this can give improved errors if something breaks or changes? Not seen it elsewhere in our codebase so a general q.

Reply:
that's a grand idea. i'll add it here :)